products/matomo.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Matomo
+category: server-app
+iconSlug: matomo
+permalink: /matomo
+alternate_urls:
+-   /piwik
+releasePolicyLink: "https://matomo.org/faq/new-to-piwik/faq_18925/"
+changelogTemplate: "https://github.com/matomo-org/matomo/releases/tag/__LATEST__/"
+versionCommand: console core:version
+releaseDateColumn: true
+activeSupportColumn: true
+
+auto:
+-   git: https://github.com/matomo-org/matomo
+    # https://regex101.com/r/FlTtCg/1
+    regex: ^(?<major>\d+)\.?(?<minor>\d+)\.?(?<patch>\d+)?\.?(?<tiny>\d+)?$
+
+# eol(x) = releaseDate(x+2) = comfirm on https://fr.matomo.org/blog/2016/01/announcing-long-term-support-in-matomo-the-analytics-platform-for-your-mission-critical-projects/
+# support(x) = releaseDate(x+1)
+releases:
+-   releaseCycle: "5"
+    releaseDate: 2023-12-18
+    support: true
+    eol: false
+    latest: "5.0.1"
+    latestReleaseDate: 2024-01-05
+
+-   releaseCycle: "4"
+    releaseDate: 2020-11-24
+    support: 2023-12-18
+    eol: 2024-12-19
+    latest: "4.16.0"
+    latestReleaseDate: 2023-11-28
+
+-   releaseCycle: "3"
+    releaseDate: 2016-12-19
+    support: 2020-11-24
+    eol: 2021-12-01
+    latest: "3.14.1"
+    latestReleaseDate: 2020-09-11
+    
+-   releaseCycle: "2"
+    releaseDate: 2013-12-17
+    support: 2016-12-19
+    eol: 2017-12-18
+    latest: "2.17.1"
+    latestReleaseDate: 2016-11-14
+
+-   releaseCycle: "1"
+    releaseDate: 2010-08-28
+    support: 2013-12-17
+    eol: true
+    latest: "1.12"
+    latestReleaseDate: 2013-05-30
+
+-   releaseCycle: "0"
+    releaseDate: 2008-04-11
+    support: 2010-08-28
+    eol: true
+    latest: "0.9.9"
+    latestReleaseDate: 2010-08-27
```

Comment:
Matomo doesn't have a clear documentation on support/eol date

V2 support date, based on this [blog post](https://matomo.org/blog/2017/12/piwik-2-reaches-end-life-soon-december-2017-update-now/)
V3 support date, baded on waybackmachine [here](https://web.archive.org/web/20220815000000*/https://fr.matomo.org/blog/2016/01/announcing-long-term-support-in-matomo-the-analytics-platform-for-your-mission-critical-projects/)
V4 support date, based on another [blog post](https://fr.matomo.org/blog/2016/01/announcing-long-term-support-in-matomo-the-analytics-platform-for-your-mission-critical-projects/)

Also, since V2, they introduce LTS releases, but, for me that's not realy "LTS" as all major version seems to be LTS.  
It seems to be a rather N-1 support strategy, where N-1 releases only receive critical bug or security patches.

The LTS part is more clearly documented [here](https://github.com/matomo-org/matomo/issues/8546), but some links mentioned in the issue are now in 404